client/src/panels/DataTable/index.ts

```diff
@@ -0,0 +1,105 @@
+import { Disposable, Uri, ViewColumn, WebviewPanel, window } from "vscode";
```

Comment:
copyright statement

Reply:
Fixed all the missing copyrights (https://github.com/sassoftware/vscode-sas-extension/pull/143/commits/eab3fdb24b9b1b0deaa27720484a879fbaa02807) and added `npm run copyright:check` to make sure this isn't happening moving forward (https://github.com/sassoftware/vscode-sas-extension/pull/143/commits/78b14e30dc925ca780db414b44da99bdb16b45c4). 

Last, I made this part of our github actions so that we theoretically won't have to worry about checking each file for copyright manually (https://github.com/sassoftware/vscode-sas-extension/pull/143/commits/33f0cffee044858cd1ca5dcd45afd7a596a4443b).

Would you take a close look at this script to make sure I'm looking for copyright information in the "correct" way - https://github.com/sassoftware/vscode-sas-extension/pull/143/files#diff-1ae1758015fed0fadbe09d992dd2d46988a9b6dde1c28ad6ef0bc32b37e73593R4? Right now, it looks like we...
 - expect copyright on code that is used as part of the extension
 - do not expect copyright on certain build files (eslintrc, build.mjs)
 - do not expect copyright on testing files

It's also worth mentioning that this check _only_ looks at files added in the PR

Reply:
Looks cool! Thanks.